static/js/lib/validation_test.js

```diff
@@ -233,6 +233,27 @@ describe("validation library", () => {
         }
       })
     })
+
+    it("should complain about an MIT email", () => {
+      [
+        ["user@mit.edu", true],
+        ["user@test.mit.edu", true],
```

Comment:
As long as they have a Kerberos account, I believe they should be able to login via Touchstone.  @pdpinch do you want to handle potential cases where someone might have an MIT email address but no Kerberos account (besides making them use a different email address)?  One option is enabling collaboration accounts.

Reply:
So we discussed this a bit in Slack. 

@csail.mit.edu users have kerberos accounts, and I'd like to encourage them to use Touchstone. 

@alum.mit.edu users do _not_ have kerberos accounts. 

I think we want to err on the side of encouraging kerberos, so I propose adding a special exception for @alum.mit.edu but otherwise forcing other .mit.edu e-mails holders to use Touchstone.